vidarr-pluginapi/src/main/java/ca/on/oicr/gsi/vidarr/api/SubmitWorkflowResponse.java

```diff
@@ -13,6 +13,11 @@
           name = "missing-key-version"),
   @JsonSubTypes.Type(value = SubmitWorkflowResponseSuccess.class, name = "success")
 })
-public abstract class SubmitWorkflowResponse {
+public abstract sealed class SubmitWorkflowResponse
```

Comment:
Why?

Reply:
It's a new feature that prevents creating unintended subclasses. For these things which are JSON, no subclasses would be allowed, so sealed.